doc/source/structures/collections/iterator.rst

```diff
@@ -83,23 +73,6 @@ Members
           - The object currently being pointed to
 
 
-.. method:: Iterator:RESET
-
-    :returns: n/a
-
-    This suffix has been deleted from kOS.
-
-    .. note::
-
-        Previous versions of kOS had a ``:RESET`` suffix for Iterators.  This doesn't
-        exist anymore and is being left in the documentation here just so people trying
-        to search for it will find this message explaining where it went.  kOS had to
-        drop it because it's no longer as easy to implement it under the hood with
-        newer versions of .Net.
-
-    (If you want to restart an iteration you must call the ``:ITERATOR`` suffix of
-    the collection again to obtain a new iterator.)
-
```

Comment:
Disagree, see above for why.

Reply:
I'm returning lines 86-90, but I think the rest is repeating what you said better just before the members table so I'm leaving it out.